From 76535fcda32c3af3e02876afb80c6b16242af9b3 Mon Sep 17 00:00:00 2001
From: Trent Mick <trentm@gmail.com>
Date: Wed, 14 Dec 2016 16:13:06 -0800
Subject: [PATCH] joyent/node-smartdc-auth#11 privateKeySigner doesn't allow
 opts.key to be a buffer Reviewed by: Alex Wilson <alex.wilson@joyent.com>
 Approved by: Alex Wilson <alex.wilson@joyent.com>

---
 lib/index.js | 1 -
 package.json | 2 +-
 2 files changed, 1 insertion(+), 2 deletions(-)

diff --git a/lib/index.js b/lib/index.js
index a9bedb3..0181835 100644
--- a/lib/index.js
+++ b/lib/index.js
@@ -80,7 +80,6 @@ function privateKeySigner(options) {
 		throw (new Error('options.key (a String or Buffer) is ' +
 		    'required'));
 	}
-	mod_assert.string(options.key, 'options.key');
 	mod_assert.string(options.user, 'options.user');
 	mod_assert.optionalString(options.subuser, 'options.subuser');
 	mod_assert.optionalString(options.keyId, 'options.keyId');
diff --git a/package.json b/package.json
index e37ff7b..b58f8b4 100644
--- a/package.json
+++ b/package.json
@@ -1,7 +1,7 @@
 {
     "name": "smartdc-auth",
     "description": "Joyent Authentication Library",
-    "version": "2.5.2",
+    "version": "2.5.3",
     "author": "Joyent (joyent.com)",
     "contributors": [
         "Mark Cavage",
-- 
2.21.0

